BuildProject/ios/BuildProject.xcodeproj/project.pbxproj

```diff
@@ -526,7 +526,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
Does this project build and run on Apple Silicon machines with this change? There used to be issues with trying to run iOS Simulator with arm64-built binaries.

If you see errors locally like the kind described in the link below, try reverting this line and line 597 of this file.

https://stackoverflow.com/questions/65978359/xcode-error-building-for-ios-simulator-but-linking-in-dylib-built-for-ios-f

Reply:
I didn't change this manually.  I'm guessing loading into xcode and saving the project changed this.